yt/data_objects/construction_data_containers.py

```diff
@@ -922,7 +922,8 @@ def __init__(self, *args, **kwargs):
         self._final_start_index = self.global_startindex
 
     def _setup_data_source(self, level_state = None):
-        if level_state is None: return
+        if level_state is None:
+            return super(YTSmoothedCoveringGrid, self)._setup_data_source()
```

Comment:
I'd format this as:

```
if level_state is None:
    super(YTSmoothedCoveringGrid, self)._setup_data_source()
    return
```

`_setup_data_source` doesn't return anything, and this way of calling it makes it clearer that that's the case.